charts/resource-injector/README.md

```diff
@@ -37,9 +37,9 @@ resources:
 
 ## Values
 
-| Key | Type | Default | Description |
```

Comment:
Excuse this diff in here. I edited a global template, hence it re-rendering for other charts too.